src/lib.rs

```diff
@@ -1,82 +1,100 @@
-//! This crate contains checked implementations of `transmute()`.
+//! This crate contains checked implementations of transmutation procedures, some of which
+//! ensure memory safety.
+//! 
+//! ## Crate outline
+//! 
+//! The following modules are available:
+//! 
+//! - The functions in the [`base`](base/index.html) module are not inherently
+//! safe, but just protected against out of boundary access (like trying to
+//! create an 8-byte type from 7 bytes). These functions are as safe as
+//! the data passed to them: any attempt of transmuting data to an invalid
+//! memory representation is still undefined behavior. Moreover, unaligned
+//! memory access is not prevented, and must be previously ensured by the
+//! caller.
+//! - The [`guard`] module contains the **Guard** API, which imposes slice
+//! boundary restrictions in a conversion.
+//! - The [`pod`] module introduces the `PodTransmutable` trait, which
+//! statically ensures that any bit combination makes a valid value for a
+//! given type. The functions in this module are safer than [`base`], but still
+//! do not prevent unaligned memory access.
+//! - `to_bytes` enables the opposite operation of reintepreting values as bytes. 
+//! - The `bool` module ensures safe transmutation of bytes to boolean values and
+//! vice versa.
+//! - At the root of this crate, there are transmutation functions with enough
+//! checks to be considered safe to use in any circumstance. The operation may
+//! still arbitrarily raise (recoverable) errors due to unaligned data, but it
```

Comment:
"return" – in keeping with stdlib docs panics are "raised", but errors are "returned".